loader/icd_dispatch.c

```diff
@@ -196,8 +202,12 @@ clCreateContextFromType(const cl_context_properties * properties,
     // initialize the platforms (in case they have not been already)
     khrIcdInitialize();
 
-    // determine the platform to use from the properties specified
+    // determine the platform to use from the properties and device_type specified
```

Comment:
Minor: This comment may have been from an earlier commit and the platform is no longer determined by the device type (if it ever was).

Reply:
Fixed.